src/app/core/component/markdown-content-container/markdown-pattern-sectioncontent/markdown-pattern-section-content.component.ts

```diff
@@ -174,26 +174,26 @@ export class MarkdownPatternSectionContentComponent extends DataRenderingCompone
   }
 
   commentSVG() {
-    const snackBarInstruction = this.snackBar.open('Mark area to comment in Picture', null, {duration: 3000});
+    const snackBarInstruction = this.snackBar.open('Mark area to comment in Picture', null, { duration: 3000 });
     this.isCommentingEnabled = true;
 
 
```

Comment:
remove unnecessary blank line

Reply:
As this code is written and maybe still in work by @mbeisel , I prefer to leave it as it is. Just fixed the spaces within the brackets for the linter.